### PR TITLE
docs: improved ngrx docs

### PR DESCRIPTION
**Pull request by SchettlerKoehler**

### CODE REVIEW

File: docs/modules/angular/pages/ngrx/ngrx.adoc [hunk 1]
```diff
@@ -39,12 +39,13 @@ NOTE: Folders should only be created and used if they have content.
 │   │   │    │    │    ├── featurecomponent1.component.ts
 │   │   │    ├── directives     // optional - directives used in specific feature module
 │   │   │    ├── pages
-│   │   │    │    ├── page1       // one of pages, might have many of them
-│   │   │    │    │     ├── page1subcomponent1    // one of subcomponent for page, optional + might have many of them
+│   │   │    │    ├── page1       // each pages gets a subfolder, might have many of them
+│   │   │    │    │     ├── page1subcomponent1    // one of subcomponent for page, optional + might have many of them, this can also be a dialog that is only used by this pages
```
Comment (BenjaminPabst) [inline docs/modules/angular/pages/ngrx/ngrx.adoc:43]: dialogs should be in a subfolder (page1/dialogs/dialog1...) to better structure the files

File: docs/modules/angular/pages/ngrx/ngrx.adoc [hunk 2]
```diff
@@ -39,12 +39,20 @@ NOTE: Folders should only be created and used if they have content.
 │   │   │    │    │    ├── featurecomponent1.component.ts
 │   │   │    ├── directives     // optional - directives used in specific feature module
 │   │   │    ├── pages
-│   │   │    │    ├── page1       // one of pages, might have many of them
+│   │   │    │    ├── page1       // each pages gets a subfolder, might have many of them
```
Comment (anninowak) [inline docs/modules/angular/pages/ngrx/ngrx.adoc:42]: each page / every page gets
oder 
all pages get

File: docs/modules/angular/pages/ngrx/ngrx.adoc [hunk 3]
```diff
@@ -57,25 +65,35 @@ NOTE: Folders should only be created and used if they have content.
 │   │   │    │    │     ├── page1.effects.spec.ts
 │   │   │    │    │     ├── page1.effects.ts
 │   │   │    │    │     ├── page1.actions.ts
+│   │   │    │    │     ├── page1.viewmodel.ts
+│   │   │    ├── dialogs // dialogs which are used across pages, optional
+│   │   │    │    ├── dialog1       // each dialog gets a subfolder, might have many of them
+│   │   │    │    │     ├── dialog1subcomponent1    // one of subcomponent for page, optional + might have many of them
```
Comment (anninowak) [inline docs/modules/angular/pages/ngrx/ngrx.adoc:71]: subcomponent for page or dialog?

File: docs/modules/angular/pages/ngrx/ngrx.adoc [hunk 4]
```diff
@@ -108,6 +129,31 @@ NOTE: Folders should only be created and used if they have content.
...
 
+== Subcomponents
+Usually subcomponents have a single input for the viewmodel called `vm`. This viewmodel is a member of the components viewmodel and is build via selectors. If a subcomponent has also a subsubcomponent, the viewmodel of the subcomponent has a member containing the viewmodel of the subsubcomponent.
+
+== Dialogs
+There are two kinds of dialogs
+    * dialogs only used in one page
+    * dialogs used in multiple pages
+
+Both dialog types have in common that
```
Comment (anninowak) [inline docs/modules/angular/pages/ngrx/ngrx.adoc:140]: that:

File: docs/modules/angular/pages/ngrx/ngrx.adoc [hunk 5]
```diff
@@ -108,6 +129,31 @@ NOTE: Folders should only be created and used if they have content.
...
+There are two kinds of dialogs
+    * dialogs only used in one page
+    * dialogs used in multiple pages
+
+Both dialog types have in common that 
+* they are opened in effects
+* the result is handled in the effects and is converted into an action
+* they have a viewmodel
+* they are opened with `PortalDialogService.openDialog(...)`
+* they are communicating with the opener of the dialog by implementing `DialogResult`
```
Comment (anninowak) [inline docs/modules/angular/pages/ngrx/ngrx.adoc:145]: Was meinst du genau mit dem Opener vom Dialog?

File: docs/modules/angular/pages/ngrx/ngrx.adoc [hunk 6]
```diff
@@ -108,6 +129,31 @@ NOTE: Folders should only be created and used if they have content.
...
+    * Implementing `DialogButtonClicked` can help setting the dialog result before the dialog closes
+
+=== Dialogs used in one page
+The components of this dialogs are subcomponents of the page they are used in. The subcomponents are placed in a subfolder called 'dialogs'. They do not have a septate section in the store because they are sharing the data of the page they are belonging to. The viewmodel can be passed via an input which is called 'vm'. The viewmodel is read from the store via a viewmodel selector in the effects or build with data from the action which is triggering the opening of the dialog.
```
Comment (anninowak) [inline docs/modules/angular/pages/ngrx/ngrx.adoc:149]: this dialog
oder 
these dialogs

Comment (anninowak) [inline docs/modules/angular/pages/ngrx/ngrx.adoc:149]: They do not have a septate section
--> separate ?

Comment (anninowak) [inline docs/modules/angular/pages/ngrx/ngrx.adoc:149]: The viewmodel is read from the store via a viewmodel selector in the effects or _is_ build with data from the action which is triggering the opening of the dialog.

File: docs/modules/angular/pages/ngrx/ngrx.adoc [hunk 7]
```diff
@@ -108,6 +129,31 @@ NOTE: Folders should only be created and used if they have content.
...
+
+The dialogs can also dispatch actions but only if they are not used to react on the closing of the dialog. Reacting to the closing is always done via the Observable returned by `openDialog(...)`
+
+=== Dialogs used in multiple pages
+The structure and inner workings of a dialog that is used in many pages is similar to the structure of a page. They have their own section in the store, own effects, own selectors,...
+This type of dialog is always dispatching an action that is informing about the opening of the dialog in the constructor of the dialog component. The viewmodel is read from the store via a selector like it is done for a page.
```
Comment (anninowak) [inline docs/modules/angular/pages/ngrx/ngrx.adoc:155]: Meinst du einen bestimmten DialogTyp hier? (mit this type of dialog)?

**Review by anninowak — CHANGES_REQUESTED** [review]
Hi Jan, anbei meine Anmerkungen.

**Pull request merged**